data/mods/BombasticPerks/perkmenu.json

```diff
@@ -355,6 +355,24 @@
         ],
         "topic": "TALK_PERK_MENU_SELECT"
       },
+      {
+        "condition": { "not": { "u_has_trait": "perk_vengeful" } },
+        "text": "Gain [<trait_name:perk_vengeful>]",
+        "effect": [
+          { "set_string_var": "<trait_name:perk_vengeful>", "target_var": { "context_val": "trait_name" } },
+          {
+            "set_string_var": "<trait_description:perk_vengeful>",
+            "target_var": { "context_val": "trait_description" }
+          },
+          { "set_string_var": "perk_vengeful", "target_var": { "context_val": "trait_id" } },
+          {
+            "set_string_var": "Requires handguns 2",
```

Comment:
This is a typo I think

Reply:
You're right--I removed it from the actual requirements but not from the listed requirements